garage/experiment/local_tf_runner.py

```diff
@@ -240,15 +340,23 @@ def train(self,
             else:
                 batch_size = 40 * self.algo.max_path_length
```

Comment:
this is a little bit strange. if there is this magic default, can I at least access it through a class property (and have it be properly documented)

Reply:
What about making `batch_size` a required parameter. I can see a case where a missing `batch_size` makes sense.

Reply:
I made `batch_size` required.